packages/mobile/src/transactions/TransferFeedItem.tsx

```diff
@@ -35,13 +35,15 @@ function navigateToTransactionReview({
   }
 
   const recipient = getRecipientFromAddress(address, addressToE164Number, recipientCache)
+  const e164PhoneNumber = addressToE164Number[address] || undefined
```

Comment:
is this just to make extra clear that it could be undefined?

Reply:
It's to force `null` to `undefined`. I don't really care about the difference between the two but all of the type checking downstream is expecting `string` or `undefined`